packages/contracts/contracts/Import.sol

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import { Forwarder } from "@thirdweb-dev/contracts/forwarder/Forwarder.sol";
```

Comment:
This just ensures the artifact is generated so it can be used during the deployment process.